content/docs/ops/bosh-releases-in-15-minutes-with-ubuntu.md

```diff
@@ -0,0 +1,143 @@
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+exec $SHELL
+
+git clone git://github.com/sstephenson/ruby-build.git ~/.rbenv/plugins/ruby-build
+echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.bashrc
+exec $SHELL
+```
+
+### Install Ruby 2.3.1
+Ruby is used for several tools including the BOSH CLI and Vagrant:
+```shell
+rbenv install 2.3.1
+rbenv global 2.3.1
+echo "gem: --no-ri --no-rdoc" > ~/.gemrc
+```
+
+### Install BOSH CLI
+BOSH has a command line client and a server it connects to.  To install the client, which will be the primary interface to BOSH: 
+```shell
+gem install bosh_cli --no-ri --no-rdoc
+```
+
+This client can speak with both BOSH lite and regular BOSH servers.
+
+### Install BOSH Lite
+BOSH Lite provides a lightweight BOSH director running on a Vagrant VirtualBox instance.
+```shell
+mkdir $HOME/workspace
+cd $HOME/workspace
+git clone https://github.com/cloudfoundry/bosh-lite
+cd bosh-lite
+vagrant up
+```
+
+### Connect to BOSH Lite
+Configure the BOSH command line client to connect to the new director running in Vagrant.  The default username is `admin` and password is `admin`:
+```shell
+bosh target 192.168.50.4 lite
+```
+
+### Add Routes
+The VirtualMachine is up and will create containers to service our BOSH requests.  In order for these containers to communicate with the network, a route needs to be added:
+
+```shell
+sudo ./bin/add-route
+```
+
+### Inspect the Director
+The director is up and running and can be inspected to review the configuration.  Currently the director is empty since nothing has been configured or uploaded:
+```shell
+bosh releases
+bosh stemcells
+bosh deployments
+```
+
+### Upload a Release
+[cg-secureproxy-boshrelease](https://github.com/18F/cg-secureproxy-boshrelease) is a BOSH release the cloud.gov team maintains and is a good example.  Clone the release, create a BOSH release and upload it to the director:
+```shell
+cd $HOME/workspace
+git clone https://github.com/18F/cg-secureproxy-boshrelease.git
+bosh create release
+bosh upload release
+```
+
+### Upload a Stemcell
+There is now a `secureproxy` BOSH release, but a operating system is needed to run it on.  These images are called `stemcells` and can be found at [bosh.io](https://bosh.io/stemcells):
+```shell
+cd $HOME/workspace
+curl -O https://s3.amazonaws.com/bosh-core-stemcells/warden/bosh-stemcell-3312.12-warden-boshlite-ubuntu-trusty-go_agent.tgz
+bosh upload stemcell ./bosh-stemcell-3312.12-warden-boshlite-ubuntu-trusty-go_agent.tgz
+```
+
+### Create a Deployment
+There is now a `secureproxy` BOSH release and a `Ubuntu Trusty` stemcell.  A deployment manifest tells the director which combination of releases, jobs and stemcells need to be deployed.  Each release usually contains a sample deployment manifest or tool to generate one:
+```shell
+cd $HOME/workspace/cg-secureproxy-boshrelease/templates
+./make_manifest warden
+```
```

Comment:
Do we have any tips / FAQs to let the user know that it was user error vs something magically wrong?